python/CMakeLists.txt

```diff
@@ -44,38 +45,27 @@ set(CUML_CPP_TARGET "cuml++")
 ################################################################################
 # - Process User Options  ------------------------------------------------------
 
-
-# If the user requested it,  we attempt to find cuml.
+# If the user requested it, we attempt to find cuml.
 if(FIND_CUML_CPP)
-  include(rapids-cpm)
-  rapids_cpm_init()
-  include(rapids-cuda)
-  rapids_cuda_init_architectures(cuml-python)
```

Comment:
We need to keep:
```cmake
  rapids_cuda_init_architectures(cuml-python)
  enable_language(CUDA)
  include("${CMAKE_PROJECT_cuml-python_INCLUDE}")
  ```
  So that if we find a cuml + raft that has cuda enabled everything works correctly

Reply:
Moved this out of the conditional block where we're building libcuml and instead doing this globally.